tests/GraphqlTest/ToggleRequestWord.txt

```diff
@@ -0,0 +1,7 @@
+mutation {
+  toggleRequest(
+    word: { literal: "Blåbærgrød" }
+  ) {
+    id
```

Comment:
Bedst tilføje med: `requesters_count` og `user_requested` for at faktatjek om tallet går op/ned og brugeren har requestet det eller ej